src/services/codefixes/importFixes.ts

```diff
@@ -684,8 +703,19 @@ namespace ts.codefix {
         }
     }
 
-    function getActionsForNonUMDImport(context: ImportCodeFixContext, allSourceFiles: ReadonlyArray<SourceFile>, cancellationToken: CancellationToken): CodeAction[] {
-        const { sourceFile, checker, symbolName, symbolToken } = context;
+    function getActionsForNonUMDImport(context: CodeFixContext): CodeAction[] {
+        // This will always be an Identifier, since the diagnostics we fix only fail on identifiers.
+        const { sourceFile, span, program, cancellationToken } = context;
+        const checker = program.getTypeChecker();
+        const symbolToken = getTokenAtPosition(sourceFile, span.start, /*includeJsDocComment*/ false);

```

Comment:
It looks like this is the same in both UMD and non-UMD cases.

Reply:
yes. but i wanted to defer the call to convertContext untill i had the name.